### ISSUE TITLE
Is this really a lint'ing related issue ?

### ISSUE THREAD

**Issue by taherv** [golang/lint#235]
So I got this error from golint ....

volume.go:47:1: comment on exported type StorageDevice should be of the form "StorageDevice ..." (with optional leading article)

My code was this ....
// The rest of the code is a work in progress.
type StorageDevice interface {
        CreateVolume() Volume
}

Why does golint think I will want to mention the interface name in the comment ?
(Isn't saying "the interface below" ... or "this interface" just as useful a comment ?

Why does golint think I will always want to comment using the English language ?

I feel being judged ! :)

**Comment by bgreezy** [comment]
I have limited experience but I believe documentation like that found on godoc.org is automatically generated by comments above structs and functions that begin with the name of that exported function or struct. That is why the comment above the exported struct is flagged that way. Check out any project on godoc.org to see examples!

**Comment by dsnet** [comment]
For one, I would avoid language like "the interface below" since any documentation front-end may end up displaying the docstring above _or_ below the block.

Secondly, it is convention from the early days of the language's early days to write docstrings starting with the identifier of what it is describing. Whether or not we like with this style, I believe it is important that the community remains consistent about these types of things.

> Why does golint think I will always want to comment using the English language ?

I believe this still works fine for other languages where the subject is allowed to start at the beginning of a sentence.
```go
// 世界很大。
func 世界() string {
    return "world is very big"
}
```

**Comment by taherv** [comment]
Thanks !
This is not a bug per-se, thanks for educating me.

Not a linguist by any means, but the "(with optional leading article)" doesn't apply to every language, does it ?

**Comment by dsnet** [comment]
You are right that leading articles are not available in all languages. I'm pretty sure that we don't support articles in French in Spanish (or other languages). If someone runs into that issue, they can file an issue and we'll handle it then.

For now, I'll close this. Thanks for the report.
